src/frontends/onnx/frontend/src/op/unsqueeze.cpp

```diff
@@ -26,6 +26,15 @@ OutputVector unsqueeze(const Node& node) {
 namespace set_13 {
 OutputVector unsqueeze(const Node& node) {
     auto inputs = node.get_ng_inputs();
+    // when onnx format is < 13 but ai.onnx > 13
+    // which results in using unsqueeze-13 but
+    // the node itself has format of unsqueeze-8
+    if (inputs.size() == 1) {
```

Comment:
This is invalid from the standard's perspective. We should not support such models.

Reply:
Good point. I reverted this change.